client/src/constants/chart.js

```diff
@@ -13,4 +13,5 @@ export const connectionStyle = {
 
 export default {
   labelFont,
+  MAX_FONT_SIZE: 12,
```

Comment:
On my machine, `MAX_FONT_SIZE` is always `undefined` when imported into the charts